Sources/Purchasing/IntroEligibilityCalculator.swift

```diff
@@ -102,18 +102,12 @@ private extension IntroEligibilityCalculator {
                 result[candidate.productIdentifier] = .unknown
                 continue
             }
-            let activeSubscriptionInGroup = (
-                candidate.subscriptionGroupIdentifier != nil &&
```

Comment:
@aboedo this is simpler, and I think it would be a valid implementation too, as a way to handle products with no group?

Reply:
yep! I love it